app/models/solution.rb

```diff
@@ -219,9 +219,10 @@ def sync_git!
   end
 
   def read_file(filepath)
-    return Solution::GenerateReadmeFile.(self) if filepath == 'README.md'
-    return Solution::GenerateHelpFile.(self) if filepath == 'HELP.md'
-    return Solution::GenerateHintsFile.(self) if filepath == 'HINTS.md'
+    return Solution::GenerateReadmeFile.(self) if filepath == Git::Exercise::SPECIAL_FILEPATHS[:readme]
+    return Solution::GenerateHelpFile.(self) if filepath == Git::Exercise::SPECIAL_FILEPATHS[:help]
+    return Solution::GenerateHintsFile.(self) if filepath == Git::Exercise::SPECIAL_FILEPATHS[:hints]
+    return git_exercise.read_file_blob(exercise.git.config_filepath) if filepath == Git::Exercise::SPECIAL_FILEPATHS[:config]
```

Comment:
I wanted to try using the special file paths here, but I don't think it becomes more readable. Maybe hardcoding is better?

Reply:
I think this is better code now. And we can make `SPECIAL_FILEPATHS` cleverer later? Like 

`if Git::Exercise::SPECIAL_FILEPATHS.readme?(filepath)`